pyproject.toml

```diff
@@ -92,3 +93,25 @@ module = [
   "traceloggingdynamic",
 ]
 ignore_missing_imports = true
+
+[tool.poe.tasks.install]
+shell = "for dir in . ni_measurementlink_generator/ examples/*/; do pushd $dir; poetry install; popd; done"
+
+[tool.poe.tasks.ni-python-styleguide]
+args = [
+  { name = "command", positional = true, required = true }
+]
+shell = "for dir in . ni_measurementlink_generator/ examples/*/; do pushd $dir; ni-python-styleguide $command; popd; done"
+
+[tool.poe.tasks.mypy]
+sequence = [
+  { cmd = "mypy -p ni_measurementlink_service" },
+  { cmd = "mypy -p ni_measurementlink_generator", cwd = "ni_measurementlink_generator" },
+  { shell = "for dir in examples/*/; do pushd $dir; mypy .; popd; done" }
+]
+
+[tool.poe.tasks.test]
+sequence = [
+  { cmd = "pytest --cov=ni_measurementlink_service -v" },
+  { cmd = "pytest --cov=ni_measurementlink_generator -v", cwd = "ni_measurementlink_generator" }
+]
```

Comment:
consider adding a
```suggestion
]

[tool.poe.tasks.all]
sequence = ["install", "ni-python-styleguide", "mypy", "test"]
```

Reply:
I added a "check" task that runs all the checks.

I don't really want to combine "install" because "install" is pretty slow.